config.md

```diff
@@ -356,7 +356,8 @@ Runtime implementations MAY support any valid values for platform-specific field
 
 ## <a name="configHooks" />Hooks
 
-Hooks allow for the configuration of custom actions related to the [lifecycle](runtime.md#lifecycle) of the container.
+Hooks allow for the configuration of custom actions related to the [lifecycle](runtime.md#lifecycle) of the container if supported by the platform.
```

Comment:
I'd rather follow the [example set for `process.user.uid` and friends][1] and make this:

```Markdown
## Linux and Solaris Hooks

For Linux- and Solaris-based systems the configuration structure supports `hooks` for configuring custom actions related to the [lifecycle](runtime.md#lifecycle) of the container.
```

Although I'd rather have [this ABNF][2] so we could say:

* **`hooks`** (object, OPTIONAL, linux solaris) …

to protect readers who skipped over this lead-in paragraph and went straight to the `hooks` list entry.

[1]: https://github.com/opencontainers/runtime-spec/blob/2ec18c02474dd49ce20aa30236959a8a80763042/config.md#linux-and-solaris-user
[2]: https://github.com/opencontainers/runtime-spec/pull/747/files#diff-958e7270f96f5407d7d980f500805b1bR43